packages/debugger/test/helpers.js

```diff
@@ -27,14 +28,14 @@ export async function prepareContracts(provider, sources = {}, migrations) {
     solc: {
       version: "0.7.4",
       settings: {
-        optimizer: {enabled: false, runs: 200},
+        optimizer: { enabled: false, runs: 200 },
         evmVersion: "constantinople"
       }
     }
   };
 
   await addContracts(config, sources);
-  let {contractNames, files} = await compile(config);
+  let { contractNames, compilations: rawCompilations } = await compile(config);
```

Comment:
Oh interesting, I didn't know you could reference `compilations` this way.

Reply:
See your other comment. :)